src/test/java/tds/exam/results/mappers/OpportunityMapperTest.java

```diff
@@ -108,7 +109,7 @@ public void shouldMapExpandableExamToOpportunity() {
 
     private void mapMockExamPagesAndItems(final List<ExamPage> examPages, final List<ExamItem> examItems) {
         // Mock/map the exam page ids from "ExamItems" to actual ExamPages.
-        for (int i = 0; i < 5; i++) {
+        for (int i = 1; i < 5; i++) {
```

Comment:
With updates to the exam client, we can no longer have a 0 index on one of the properties.